komorebi/src/container.rs

```diff
@@ -1,18 +1,26 @@
 use std::collections::VecDeque;
 
+use getset::CopyGetters;
 use getset::Getters;
+use getset::MutGetters;
+use getset::Setters;
 use nanoid::nanoid;
 use serde::Deserialize;
 use serde::Serialize;
 
 use crate::ring::Ring;
 use crate::window::Window;
+use crate::Lockable;
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, Getters)]
+#[derive(
+    Debug, Clone, PartialEq, Serialize, Deserialize, Getters, CopyGetters, MutGetters, Setters,
+)]
 #[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
 pub struct Container {
     #[getset(get = "pub")]
     id: String,
+    #[getset(get_copy = "pub", get_mut = "pub", set = "pub")]
+    locked: bool,
```

Comment:
Can you look into serde annotations which will make this default to false if it is not present when serializing/deserializing? We should try to make sure that we don't break integrations by introducing new mandatory fields

Reply:
Sorry, I forgot to address this earlier!
I’ve now added `#[serde(default)]` to the locked field, so it defaults to false when missing during deserialization.